examples/integration_test.py

```diff
@@ -1263,6 +1253,7 @@ def verify_avro_explicit_read_schema():
         conf = copy(cons_conf)
         if schema_registry_url:
             conf['schema.registry.url'] = schema_registry_url
+            conf['enable.partition.eof'] = True
```

Comment:
this should be applied for both clients, not just for when a real SR is available.

Reply:
see commit message about this being removed in #440. I will be rebasing shortly

Reply:
post fast-forward:

```bash
git grep "default.topic.config"
confluent_kafka/src/confluent_kafka.c:        if ((vo = PyDict_GetItemString(confdict, "default.topic.config"))) {
confluent_kafka/src/confluent_kafka.c:                             "default.topic.config has being deprecated, "
confluent_kafka/src/confluent_kafka.c:                             "set default topic configuration values in the global dict");
confluent_kafka/src/confluent_kafka.c:                PyDict_DelItemString(confdict, "default.topic.config");
docs/index.rst:* ``default.topic.config``: value is a dict of client topic-level configuration
tests/test_misc.py:    # *NOTE* default.topic.config has been deprecated.
tests/test_misc.py:    confs = [{"message.timeout.ms": 600000, "default.topic.config": {"message.timeout.ms": 1000}},
tests/test_misc.py:             {"default.topic.config": {"message.timeout.ms": 1000}}]
```